### PR TITLE
make stack size matching optional

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Review by rom1504 — CHANGES_REQUESTED** [review]
Please add to doc

File: index.js [hunk 1]
```diff
@@ -32,15 +32,15 @@ function loader (version) {
...
+    static equal (item1, item2, matchStackSize = true) {
       if (item1 == null && item2 == null) {
         return true
       } else if (item1 == null) {
         return false
       } else if (item2 == null) {
         return false
       } else {
-        return item1.type === item2.type &&
+        return (matchStackSize ? item1.type === item2.type : true) &&
```
Comment (rom1504) [inline index.js:43]: This is not doing the right thing.
I guess you mean the below line

Comment (u9g) [reply index.js]: fixed

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
Feel free to do a release pr

**Pull request merged**